### ISSUE TITLE
Image alt text is missing from icons

### ISSUE THREAD

**Issue by emma-sg** [emma-sg/roleypoly#10]
The `alt` attribute should either specify a description of the image, or should be set to an empty string to indicate that the image does not have semantic value.
